### PR TITLE
Npm scripts

### PR DESCRIPTION
**Pull request by dunckr**
Add npm scripts so don't have to mess around with aliases.

2.0 is looking 👍

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,12 @@
     "type": "git",
     "url": "git://github.com/greypants/gulp-starter.git"
   },
+  "scripts": {
+    "watch": "./node_modules/.bin/gulp",
+    "production": "./node_modules/.bin/gulp production",
+    "iconFont": "./node_modules/.bin/gulp iconFont",
+    "karma": "./node_modules/.bin/karma"
+  },
```
Comment (shinnn) [inline package.json]: `./node_modules/.bin/` is not needed.

https://docs.npmjs.com/cli/run-script:

> n addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.

You can rewrite them into:

```
"scripts": {
  "watch": "gulp",
  "production": "gulp production",
  "iconFont": "gulp iconFont",
  "karma": "karma"
}
```

**Comment by dunckr** [comment]
@shinnn nice!! Updated with shorthand paths.

Looks like karma is slightly different and binary is under ```./node_modules/karma/bin/karma```

**Comment by greypants** [comment]
hmmm... not a bad idea... not bad. I feel a little weird though, having a project called "gulp-starter" but not using the `gulp` command. I'll have to think on this.

**Comment by greypants** [comment]
Good call. Addressed in #164 Thanks!

**Pull request closed without merging**